### ISSUE TITLE
client.Execute() response 500 command not understood

### ISSUE THREAD

**Issue by sslyle** [robinrodricks/FluentFTP#358]
The is C# running in debug mode in Visual Studio on Windows 10 Pro.

This execute assigns FtpReply = 500 command not understood
`
String cmd = "delete /MyFile.txt";
FtpReply stat = client.Execute(cmd)
`

But if I open a Windows cmd console, connect using the default ftp client in Windows, and run the same command their are no problems, the file get deleted form the FTP server.

What could be wrong, done, that I can help?

Thanks.

**Comment by sslyle** [comment]
So I when and read your code.
the first thing I found what your `client.Delete(<file>)` does little more than call `client.Execute("DELE <file>);`
So I adapted to my system structure the command to use `DELE` instead of `delete`.  `delete` works on my ftp server.  So does `DELE`.  `DELE` is not documented in the server's help.
I'm not understanding what cases this conflict but at least I have a work around.
